presto-docs/src/main/sphinx/language/decimal_limitations.rst

```diff
@@ -0,0 +1,15 @@
+
+=============================
+Decimal data type limitations
+=============================
+
+ #. Currently only basic arithmetic operators and inequality operators are supported.
```

Comment:
Could you please mention explicitly a few things that are NOT supported? E.g., does SELECT SUM(decimal_col) FROM decimal_table WHERE ROUND(decimal_col)>10 work?